docs/deployment/cognito/README.md

```diff
@@ -285,6 +294,9 @@ In this section, we will be creating certificate to enable TLS authentication at
 
         # Envoy filter
         kustomize build awsconfigs/common/aws-istio-envoy-filter/base | kubectl apply -f -
+
+        # AWS Authservice
+        kustomize build awsconfigs/common/aws-authservice/base | kubectl apply -f -
```

Comment:
Shall we combine these two services under one and called `aws-authservice` and then have `envoy-filter` and `logout-service` similar to what is has [here](https://github.com/arrikto/oidc-authservice/tree/master/e2e/manifests/authservice/base)

Reply:
Yeah I think that makes sense given that the `envoy-filter` is a part of the authorization workflow

Reply:
Will do.